cloudomate/util/recaptchasolver.py

```diff
@@ -0,0 +1,142 @@
+import json, requests
```

Comment:
I think the same comments as I've written for the `captchasolver.py` file apply here (variable naming, subclassing from `object`, newlines etc).